src/Symfony/Component/Serializer/Normalizer/ObjectNormalizer.php

```diff
@@ -25,6 +25,8 @@
  * Converts between objects and arrays using the PropertyAccess component.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
+ *
+ * @final since Symfony 6.3
```

Comment:
@nicolas-grekas I'm puzzled about this one as it fits into the category of the one we wanted to make final but it is actually extending in the tests.
Also IIRC ApiPlatform also extends it

Reply:
when it's "since Symfony 6.3" it means we need to keep bc with child classes

Reply:
yes.
what I mean is that I don't see how we can achieve what is done in 7.0 once this class is final.
so I'm wondering if this makes sense to make it final in the first place?

Reply:
however, if ApiPlatform extends it, it might make it hard for them to add support for Symfony 7.0. So the reason why it is extended should probably be investigated.

Reply:
what do you think of removing the change for this specific class and opening another PR for this one where we can discuss about the test cases and ApiPlatform with @dunglas ?

Reply:
let's do that yes

Reply:
done, see #49953